### PR TITLE
(tlg0012.tlg001)-CTS file and header revisions-

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated CTS file
Corrected and updated headers for the Iliad
Per issue #1279

### CODE REVIEW

File: data/tlg0012/tlg001/__cts__.xml [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
+      <ti:description xml:lang="eng">Homer. The Iliad of Homer, rendered into English prose for the use of those who cannot read the
+         original. Butler, Samuel, 1835-1902, translator.  London: Longmans, Green, and Co, 1898.</ti:description>
    </ti:translation>
         
    <ti:edition urn="urn:cts:greekLit:tlg0012.tlg001.perseus-grc2" workUrn="urn:cts:greekLit:tlg0012.tlg001">
-     <ti:label xml:lang="eng">Iliad, Homeri Opera</ti:label>
-     <ti:description xml:lang="eng">Homer, creator; Monro, D. B. (David Binning), 1836-1905, creator; Monro, D. B. (David Binning), 1836-1905, editor; Allen, Thomas W. (Thomas William), b. 1862, editor</ti:description>                              
+      <ti:label xml:lang="grc">Ἰλιάς</ti:label>
+      <ti:description xml:lang="mul">Homer.  Homeri Opera, Volumes 1-2. Monro, D. B. (David Binning), editor; Allen, Thomas W. (Thomas William), editor.  Oxford: e typographeo Clarendoniano, 1908-1920.</ti:description>                              
    </ti:edition>
```
Comment (lcerrato) [inline data/tlg0012/tlg001/__cts__.xml:21]: We can just use `Oxford: Clarendon Press` here

File: data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml [hunk 2]
```diff
@@ -32,12 +34,20 @@
...
-                        <title>Iliadis (Iliad) in two volumes. David B. Monro and Thomas W. Allen, eds. Editio Tertia (Third Edition).</title>
+                        <title xml:lang="lat">Homeri Opera</title>
+                        <editor>David B. Monro</editor>
+                        <editor>Thomas W. Allen</editor>
+                        <edition xml:lang="lat">Editio Tertia</edition>
                         <imprint>
-                            <publisher>Oxford, Oxford University Press</publisher>
-                            <date>1920</date>
+                            <pubPlace>Oxford</pubPlace>
+                            <publisher xml:lang="lat">E Typographeo Clarendoniano</publisher>
```
Comment (lcerrato) [inline data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml:43]: Clarendon Press

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml:43]: I've changed this and will update the pull request! On to the Odyssey!

Comment (lcerrato) [reply data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml:43]: Please add the "Press": I think we use that elsewhere. I can't add to this file online.

**Review by lcerrato — CHANGES_REQUESTED** [review]
@AlisonBabeu 
We use Clarendon Press elsewhere, so I would be sure to use that instead of just Clarendon. I changed the cts file but can't change the full text file online.

File: data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml:43]: Oops, I will do that! Sorry!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**